PAWSUITests/AccountCreationTests.swift

```diff
@@ -17,6 +17,9 @@ final class AccountCreationTests: XCTestCase {
         continueAfterFailure = false
 
         let app = XCUIApplication()
+        Task {
+            await setupSnapshot(app)
+        }
```

Comment:
We could change this method to use the async overload [`setUp()`](https://developer.apple.com/documentation/xctest/xctest/3856481-setup) to ensure that we don't need to spawn a task here that might not even complete before the function is finished executing.